dashboard/app/helpers/levels_helper.rb

```diff
@@ -570,6 +570,9 @@ def blockly_options
     level_options['puzzle_number'] = script_level ? script_level.position : 1
     level_options['lesson_total'] = script_level ? script_level.lesson_total : 1
     level_options['final_level'] = script_level.final_level? if script_level
```

Comment:
- [ ] remove `final_level` and update code referencing it to use more explicitly named `lastLevelInScript`